manifests/v1beta1/components/controller/service.yaml

```diff
@@ -19,5 +19,8 @@ spec:
     - name: metrics
       port: 8080
       targetPort: 8080
+    - name: healthz
+      port: 18080
+      targetPort: 18080
```

Comment:
```suggestion
```

I think that we do not have to add a port for probes since probes are executed by kubelet, not use kube-proxy.

Reply:
That makes sense! Thanks for finding this :)